src/terminal/RenderBufferBuilder.cpp

```diff
@@ -461,16 +462,15 @@ ColumnCount RenderBufferBuilder<Cell>::renderUtf8Text(CellLocation screenPositio
         //            unicode::convert_to<char>(u32string_view(graphemeCluster)).size(),
         //            unicode::convert_to<char>(u32string_view(graphemeCluster)));
 
-        output.cells.emplace_back(
-            makeRenderCellExplicit(terminal.colorPalette(),
-                                   graphemeCluster,
-                                   width,
-                                   textAttributes.flags,
-                                   fg,
-                                   bg,
-                                   textAttributes.underlineColor,
-                                   baseLine + screenPosition.line,
-                                   screenPosition.column + ColumnOffset::cast_from(columnCountRendered)));
+        output.cells.emplace_back(makeRenderCellExplicit(terminal.colorPalette(),
+                                                         graphemeCluster,
+                                                         width,
+                                                         textAttributes.flags,
+                                                         fg,
+                                                         bg,
+                                                         textAttributes.underlineColor,
+                                                         baseLine + screenPosition.line,
+                                                         screenPosition.column));
```

Comment:
i think this is just a reformat, isn't it? I was trying to figure what the difference here is, but seems like it's only changing the formatting, right?

Reply:
EDIT: Ah, no, the last line actually changed. wasn't that obvious from the start.

The column offsetting was removed and instead put to the top.